### PR TITLE
Nodejs week2  homework.Please review.

### PR DESCRIPTION
**Pull request by mahieakhtar**

### CODE REVIEW

File: nodejs/week1/mealSharingWebsite/src/backend/index.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                meal.reviews.push(review);
+            }
+        };
+    });
+    res.json(meals);
+    next()
+});
+//Respond with the json for all the meals (including it's reviews) that are cheap (you define what a cheap meal is)
+app.get("/cheap-meals", (req, res, next) => {
+    const cheapMeals = meals.filter(meal =>
```
Comment (benna100) [inline nodejs/week1/mealSharingWebsite/src/backend/index.js]: Great naming

File: nodejs/week2/mealSharingWebsite/src/backend/routes/meals.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    res.send(mealsData);    
+});*/
+route.get("/meals", (req, res) => {
+    res.json(mealsData);
+    
+});
+//Respond with the json for the meal with the corresponding id
+route.get("/:id", (req, res) => {
+	const { id } = req.params;
+	const mealsId = meals.filter(meal => {
```
Comment (benna100) [inline nodejs/week2/mealSharingWebsite/src/backend/routes/meals.js]: You can use array.find here

File: nodejs/week2/warmUp/index.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const route = express();
+
+route.get("/numbers/add", (req, res) => {
+	const first = Number(req.query.first);
+	const second = Number(req.query.second);
+	res.send(`${first + second}`);
```
Comment (benna100) [inline nodejs/week2/warmUp/index.js]: nice :)

File: nodejs/week1/mealSharingWebsite/src/backend/index.js (hunk 1 above)
Comment (mahieakhtar) [reply nodejs/week1/mealSharingWebsite/src/backend/index.js]: Thank you

File: nodejs/week2/warmUp/index.js (hunk 3 above)
Comment (mahieakhtar) [reply nodejs/week2/warmUp/index.js]: Thank you

**Comment by mahieakhtar** [comment]
Refactor after review.

**Pull request merged**